core/src/templates/components/quote/quote.twig

```diff
@@ -15,9 +16,12 @@
  */
 #}
 <figure {{ attributes }} class="su-quote {{ modifier_class }}">
-  {%- if src is not empty -%}
-  <img class="su-quote__img" src="{{ src }}" alt="{{ alt }}">
-  {%- endif -%}
+  {%- block block_quote_media -%}
```

Comment:
Actually could this use the nested media.twig template, like what we did for the Card component?

Reply:
Oh, that might be an option. Let me see about that refactor.